premake5.lua

```diff
@@ -144,4 +151,6 @@ project "Sandbox"
 	filter "configurations:Dist"
 		defines "HZ_DIST"
 		runtime "Release"
-		optimize "on"
+		optimize "On"
```

Comment:
why did you change this line? Doesn't it work in all small caps? In that case `"configurations:Release"` has the same issue? Either one of those requires changes then...

Reply:
No idea why I did that. Reverted